Formula/tarantool.rb

```diff
@@ -1,10 +1,10 @@
 class Tarantool < Formula
```

Comment:
The unversioned formula MUST always be the newest stable version.

Reply:
Ok, I've changed the main Formula/tarantool.rb to 2.1.2 stable version (in previous it was mistakenly set to 2.2.1 which is in real not stable but beta version, so it need to be changed to 2.1.2 which is really stable version) and added additional Formula/tarantool@1.10.rb for 1.10.4 LTS version.